### PR TITLE
feat(dns-zone): add capabilities for dynHost

### PR DESCRIPTION
**Pull request by marie-j**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/web-ca`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-5091
| License          | BSD 3-Clause

## Description

add capabilities for dynHost

### CODE REVIEW

File: packages/manager/tools/component-rollup-config/src/plugins/translation-ui-router.ts [hunk 1]
```diff
@@ -20,14 +20,16 @@ const removeProperty = (code, magicString, start, end) => {
   }
```
Comment (marie-j) [inline packages/manager/tools/component-rollup-config/src/plugins/translation-ui-router.ts:20]: Updated on `develop` but target branch `feat/web-ca` needs to be rebased. Only purpose of updating it in  this PR is to be able to check that branch builds and that behaviour is correct on a prod environment

**Comment by ovh-cds** [comment]
CDS Report build#12153.0 ✘  
* 
  * Build ✘

**Comment by ovh-cds** [comment]
CDS Report build#12153.1 ✘  
* 
  * Build ✘

File: packages/manager/tools/component-rollup-config/src/plugins/translation-ui-router.ts [hunk 2]
```diff
@@ -92,10 +91,14 @@ export = (opts:any = {}) => {
...
+              inject = `translations: /* @ngInject */ ($q, $translate, asyncLoader) => { ${inject} }`;
+
 
-              removeProperty(code, magicString,
-                translations.start, translations.end);
+                code,
+               eProperty(code, magicString,
+                tran
+                translations.end,
+              ranslations.end);
```
Comment (JeremyDec) [inline packages/manager/tools/component-rollup-config/src/plugins/translation-ui-router.ts:101]: Typos: `tran`...`ranslations.end`?

File: packages/manager/tools/component-rollup-config/src/plugins/translation-ui-router.ts [hunk 3]
```diff
@@ -92,7 +92,7 @@ export = (opts:any = {}) => {
                 format,
               );
 
-              inject = `translations: ($q, $translate, asyncLoader) => { ${inject} }`;
+              inject = `translations: /* @ngInject */ ($q, $translate, asyncLoader) => { ${inject} }`;
```
Comment (marie-j) [inline packages/manager/tools/component-rollup-config/src/plugins/translation-ui-router.ts:95]: Updated on develop but target branch feat/web-ca needs to be rebased. Only purpose of updating it in this PR is to be able to check that branch builds and that behaviour is correct on a prod environment

**Comment by marie-j** [comment]
Rebased on `feat/web-ca` and removed temporary workaround for translation injection

**Pull request merged**